### PR TITLE
Season methods

### PR DESCRIPTION
**Pull request by NSMcEnroe**
Some of the methods have been completes, but all of the hard logic steps should be figured out.  I can push the completed methods hopefully within the hour.  It'll definitely need a refactor, because a lot of this is copy and paste.

### CODE REVIEW

File: coverage/.last_run.json
Comment (reedhillmar) [inline coverage/.last_run.json:1]: coverage file, deleting because doesn't need to be tracked

File: lib/season_stats.rb [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  
+  @most_tackles_team = @team_tackles.max_by do |team_id, tackles|
+    tackles
+  end
+
+  @most_tackles_team_name = @teams.teams.find do |team|
+    team.team_id == @most_tackles_team[0]
+  end.team_name
+
+  #Later print a message that mentions goals
```
Comment (reedhillmar) [inline lib/season_stats.rb:206]: I like this a lot for a refactor: "[This team name] scored the most goals this season with [this many goals]"

**Review by reedhillmar — APPROVED** [review]
We all reviewed this together. Some good ideas for refactor and strong structure for finishing up Season methods.

**Comment by reedhillmar** [comment]
All looks good, but need to resolve conflicts outside of github. Let's have you resolve on your end then re-submit.

**Review by reedhillmar — CHANGES_REQUESTED** [review]
resolve conflicts in VSCode then resubmit. Everything looks good besides

**Comment by NSMcEnroe** [comment]
Updated the branch, requesting another PR

**Pull request closed without merging**